server/controllers/cart/deleteProductController.js

```diff
@@ -0,0 +1,15 @@
+const { deleteFromCart } = require('../../database/queries/cart');
+
+module.exports = (req, res, next) => {
+  const { user_id } = req.dataUser;
+  const { productId } = req.params;
```

Comment:
first, you need to check If the user who has this cart is the same who sent the request